pipelines/resources/deploy-ztp-edgeclusters-minimal.yaml

```diff
@@ -0,0 +1,291 @@
+    taskRef:
+      name: edgecluster-deploy-lso
+    params:
+      - name: edgeclusters-config
+        value: $(params.edgeclusters-config)
+      - name: kubeconfig
+        value: $(params.kubeconfig)
+      - name: ztp-container-image
+        value: $(params.ztp-container-image)
+      - name: mock
+        value: $(params.mock)
+      - name: pipeline-name
+        value: $(context.pipelineRun.name)
+    runAfter:
+      - deploy-workers
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+  # Deploy ODF
+  - name: deploy-odf
+    taskRef:
+      name: edgecluster-deploy-odf
+    params:
+      - name: edgeclusters-config
+        value: $(params.edgeclusters-config)
+      - name: kubeconfig
+        value: $(params.kubeconfig)
+      - name: ztp-container-image
+        value: $(params.ztp-container-image)
+      - name: mock
+        value: $(params.mock)
+      - name: pipeline-name
+        value: $(context.pipelineRun.name)
+    runAfter:
+      - deploy-lso
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+  # Deploy Disconnected Registry
+  - name: deploy-disconnected-registry
+    taskRef:
+      name: edgecluster-deploy-disconnected-registry-edgeclusters
+    params:
+      - name: edgeclusters-config
+        value: $(params.edgeclusters-config)
+      - name: kubeconfig
+        value: $(params.kubeconfig)
+      - name: ztp-container-image
+        value: $(params.ztp-container-image)
+      - name: mock
+        value: $(params.mock)
+      - name: pipeline-name
+        value: $(context.pipelineRun.name)
+    runAfter:
+      - deploy-odf
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+    # Deploy Disconnected Registry Pull secret
+  - name: deploy-disconnected-registry-ps
+    taskRef:
+      name: edgecluster-deploy-disconnected-registry-edgeclusters-ps
+    params:
+      - name: edgeclusters-config
+        value: $(params.edgeclusters-config)
+      - name: kubeconfig
+        value: $(params.kubeconfig)
+      - name: ztp-container-image
+        value: $(params.ztp-container-image)
+      - name: mock
+        value: $(params.mock)
+      - name: pipeline-name
+        value: $(context.pipelineRun.name)
+    runAfter:
+      - deploy-disconnected-registry
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+  # Deploy ICSP Edge-clusters Post
+  - name: deploy-icsp-edgeclusters-post
+    taskRef:
+      name: edgecluster-deploy-icsp-edgeclusters-post
+    params:
+      - name: edgeclusters-config
+        value: $(params.edgeclusters-config)
+      - name: kubeconfig
+        value: $(params.kubeconfig)
+      - name: ztp-container-image
+        value: $(params.ztp-container-image)
+      - name: mock
+        value: $(params.mock)
+      - name: pipeline-name
+        value: $(context.pipelineRun.name)
+    runAfter:
+      - deploy-disconnected-registry-ps
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+  # Deploy GPU Operator
+  - name: gpu-operator
```

Comment:
mmh, this probably won't work since the cluster is pointing to the internal registry, which doesn't have the images. Did it work for you?

Reply:
and...
`
tkn pr ls
I1007 00:53:06.117845   64393 request.go:665] Waited for 1.165846483s due to client-side throttling, not priority and fairness, request: GET:https://api.ocp-ztp-factory-hub-0.qe.lab.redhat.com:6443/apis/apps.ope
n-cluster-management.io/v1beta1
NAME                                        STARTED       DURATION   STATUS
deploy-ztp-edgeclusters-minimal-run-qn8f9   2 hours ago   1h23m34s   Succeeded
deploy-ztp-edgeclusters-run-fwtkr           5 hours ago   2h45m0s    Cancelled(PipelineRunCancelled)
deploy-ztp-hub-run-89ngn                    7 hours ago   1h34m16s   Succeeded
deploy-ztp-hub-run-hmhgw                    7 hours ago   6m2s       Failed

`